presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -514,6 +515,24 @@ void PrestoServer::yieldTasks() {
   }
 }
 
+void PrestoServer::initializeThreadPools() {
+  auto* systemConfig = SystemConfig::instance();
+  driverExecutor_ = std::make_shared<folly::CPUThreadPoolExecutor>(
+      systemConfig->numQueryThreads(),
+      std::make_shared<folly::NamedThreadFactory>("Driver"));
+
+  httpServiceExecutor_ = std::make_shared<folly::CPUThreadPoolExecutor>(
+      systemConfig->numQueryThreads(),
+      std::make_shared<folly::NamedThreadFactory>("HTTPService"));
+
+  const int32_t numSpillThreads = systemConfig->numSpillThreads();
+  if (numSpillThreads > 0) {
+    spillerExecutor_ = std::make_shared<folly::IOThreadPoolExecutor>(
+        numSpillThreads,
+        std::make_shared<folly::NamedThreadFactory>("Spiller"));
+  }
```

Comment:
Where we initialize http server io executor? Thanks!

Reply:
in HttpServer.cpp